_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,145 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) that has [required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+### Expectation
+
+Each target element is [owned](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element) by an element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has a [semantic role](#semantic-role) matching one of the [required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
```

Comment:
The definition os "owned" in ARIA is a bit broken. Any descendent is an owned element. That doesn't mean its treated as part of a group. The problem is that elements need to be counted. There are at least three ways an element will be considered part of a group:

- It's a direct child of the context element
- It's referenced with aria-owns from the context element
- It's a descendent of the context element, and all elements between it and the context element have role=presentation or role=none

There might be a fourth, I'm not sure about it. DL elements in HTML 5.2 allow div elements in between it and the DT/DD. I'm not entirely sure if that div counts as an implicit group or not.

Reply:
@WilcoFiers: At least some AT follows the specification. I would say it is just a question of accessibility support - and there is already a note.

Reply:
@WilcoFiers if I understand correctly, you are describing how some ATs behave rather than what the spec says?